### ISSUE TITLE
On error, currency question focus outline doesn't have the right gap

### ISSUE THREAD

**Issue by shreyachatterjee00** [civiform/civiform#9385]
The outline of a box should have a gap so that it is legible. The gap adds enough contrast to solve the accessibility issue. Fix the text question so that the gap exists.

![image](https://github.com/user-attachments/assets/1f7228db-bc30-4f13-87a4-9332552690ae)

vs. 

![image](https://github.com/user-attachments/assets/09031dff-dcbe-4e14-97f0-8d2065ab6e19)

**Comment by melanie-exygy** [comment]
This is an accessibility error because the color contrast ration between the focus outline and the red border isn't high enough.  The contrast ration is 2.18.
